### ISSUE TITLE
As a user I need to be able to see what the name of the forest is on all pages so that I can be clear where I am.

### ISSUE THREAD

**Issue by DawnDesigner** [18F/fs-open-forest-platform#567]
## Notes
* As a user I need to be able to see what the name of the forest is on all pages so that I can be clear where I am. *

## Acceptance Criteria
- [x] Title in mock up meets 508 standards with hierarchy
- [] Consideration for future title is added to forest name for potential linking between forests (potential issues with where in the other forest the user should land if they chose a different forest from a form page)
- [x] Tan background color #E2DCC9 (medium tan) - Title text is #333333 (dark grey)
- [x] Applicants can see what forest they are applying for throughout the application process.

## Tasks
- [x] Page meets mock up criteria
- [x] PO approved

## Definition of Done
- [x] Pull requests meet technical definition of done
- [ ] Compare finished design with mock up
- [ ] Usability tested

![MtBakLanding_Title](https://user-images.githubusercontent.com/47989077/54389238-0544f000-4676-11e9-8a38-d09151b24371.png)
![MtBakPermit_Title](https://user-images.githubusercontent.com/47989077/54389242-070eb380-4676-11e9-9267-9d95b48af8ca.png)

**Comment by DawnDesigner** [comment]
Amber comment from #533  

 "Not sure if we want "Open Forest Permitting" on each page, or something different, perhaps just "Open Forest" somewhere discreetly"

Thanks Amber -  This individual story actually refers to the name of the forest directly under the top banner in the mock up above being on all pages of the application.  We all felt that placement worked.

Finding a way to include 'Open Forest' to all the sites pages is another discussion we'd love your input on. Story not yet created.

**Comment by ASprinkle** [comment]
from meeting:
Drop down works on landing page. 
Subsequent pages would not have a forest selection drop down.

**Comment by mtlaney** [comment]
Just to be clear, this issue is only referencing the new forest title section, and not the new mockups of the home page?

Also, it appears to be an h2 in the mockup, is that correct? For 508 do we want the h1 to be "Apply for a temporary outfitters permit with Open Forest." ?

**Comment by carlsonem** [comment]
@DawnDesigner the team mentioned you had an updated mock-up. If so, can you please provide? Mike asked for some clarity on what specifically needs to be done for this issue. Might be good for a quick chat.

**Comment by DawnDesigner** [comment]
To clarify, on the individual permitting application landing page (as an example Christmas Tree) if there is more than one forest, the name of the forest at the top of the page will be a drop down with multiple forest.

All other pages of the application (as example application form pages) will not have the opportunity to click to another forest to prevent confusion for the user.

Font size should be H2, and background color should be #E2DCC9, 60 pixels high, 100% wide. Should show on all device.

This will be a standard design feature on all Open Forest permit sites

Thanks

![ForestNameWithTildie](https://user-images.githubusercontent.com/47989077/55988536-1af80600-5c72-11e9-84bb-a9a7fa8270b9.png)


all other pages
![Name of forest added](https://user-images.githubusercontent.com/47989077/55988543-221f1400-5c72-11e9-80a9-c1f368fc2b1c.png)

**Comment by mtlaney** [comment]
@DawnDesigner 
This is how it looks on multiple select dropdown pages, do we still want the dropdown list? It seems a bit confusing with select forest options.
<img width="1251" alt="Screen Shot 2019-04-15 at 11 03 56 AM" src="https://user-images.githubusercontent.com/18233188/56143306-4b8bb880-5f6e-11e9-9fcf-564a3622016b.png">

**Comment by jdamis4XD** [comment]
@DawnDesigner I agree the dropdown to the forest is redundant on this page.  Could the choose your forest field not support all forests in the future?  Also, if the forest options become more than 10 the selection process will be daunting for a user and I would recommend another component like an auto complete search field might work better.   See this report about drop down accessibility options to consider: https://baymard.com/blog/drop-down-usability

**Comment by mtlaney** [comment]
@DawnDesigner @ASprinkle @jdamis4XD Ok, for the time being I'm going to push the code without the dropdown, but with the additional forest titles so we can get some traction.

**Comment by jdamis4XD** [comment]
@mtlaney That works.

**Comment by jdamis4XD** [comment]
@DawnDesigner Could you split the drop down feature into a separate story if it's not already?

**Comment by DawnDesigner** [comment]
@ASprinkle @jdamis4XD @mtlaney 
Amber, it was decided that the drop down at the top where the title of the forest  is might be confusing for users because of it's placement - and because that text is being used as a title on all other pages possibly creating confusion for the user. 

Since we have agreed there would be a hero at the top of each permit landing page (like the Xmas Tree), we would like to suggest we keep the drop down box in the hero text area like it's now being displayed on the Christmas Tree permitting landing page as a solution. 

If this solution works for Amber, there would be no need to break this into two stories unless Amber would like further discussion on where to display this choice for the user. We can establish this as a standard feature on all permit landing pages.

Thanks Mike, once the title is completed and this is solution is approved by Amber this issue can be closed unless someone has another suggestion.

**Comment by jdamis4XD** [comment]
@DawnDesigner @mtlaney @ASprinkle  For the long term as more forests are added as options will the drop down component work or will it have to be reworked? I am not sure what the final count of forests would be for the list, but having a screen reader go through more than 10 options in a list is not an ideal for a non-sighted person.

**Comment by ASprinkle** [comment]
@jdamis4XD great point.  For purposes of this project, we're focused on MVP to launch the pilot for special uses.  We used the drop down feature for Christmas trees as well. I do not think the drop down selection for National Forests is the long term solution for a couple of reasons (and now including what you mention!).  For purposes of MPV for the MBS special uses pilot launch, a drop down may suffice.

**Comment by carlsonem** [comment]
@mtlaney any update on this?

**Comment by davemcorwin** [comment]
@DawnDesigner @lisaredux @jdamis4XD 

Now that I've seen the PR, I have a couple questions:

1. The color specified for this new header does not exist in our theme and doesn't seem visually different (to my eyes) than our default black `#212121`, is it necessary?
2. This was implemented using an `h2` element which brings up some semantic concerns on certain pages where we end up with `h2`, `h1`, `h2`, in that order, should this implemented as a different element?

**Comment by jdamis4XD** [comment]
@davemcorwin @DawnDesigner @ASprinkle As pertains to #2 the header tags should be used in hierarchical order so you always start with h1 and then can have numerous h2's.  If the forest name is the page header it should be an h1.

**Comment by DawnDesigner** [comment]
We will be  adding the "Open Forest" title to the top banner which will be H1.  If this accessibility isn't counting the top banner as being a part of the hierarchical flow it can be H1 otherwise please leave it as H2.  Does that make sense?

That addition has not yet become an issue but is on Trello at @MelissaBraxton suggestion so we can be sure we don't need to get re-approval from USDA on the top banner.

**Comment by DawnDesigner** [comment]
Forest name will be H1 until we know how the name Open Forest in the top banner will be handled.

As a temporary solution, Mike will be developing the forest names in H1 until we are clear on the top banner needing approval from USDA (may take some time), and design team discusses using an image versus text for the name Open Forest name.  If we go with text Mike says changing the forest name to H2 isn't a heavy lift.

**Comment by davemcorwin** [comment]
@DawnDesigner @mtlaney Do we want to remove the extra forest name on the xmas trees pages as well?

![Screen Shot 2019-04-22 at 11 22 44 AM](https://user-images.githubusercontent.com/6662371/56517168-27d9ec80-64f1-11e9-8dec-8b14fd29244d.png)
![Screen Shot 2019-04-22 at 11 22 28 AM](https://user-images.githubusercontent.com/6662371/56517169-27d9ec80-64f1-11e9-8e3a-68140575c22e.png)

**Comment by ASprinkle** [comment]
@davemcorwin @DawnDesigner @mtlaney Let's hold off on putting the forest name header on the Christmas tree pages for now.  This story is focused on Special Uses.  We with more Usability Testing we will know about whether or not this is a path we will continue to take.

**Comment by ASprinkle** [comment]
The Forest Name appears to be showing up on each page per the design mock ups.

![image](https://user-images.githubusercontent.com/33435621/57102778-24a4e480-6cd9-11e9-8371-f2a4185cb71d.png)

![image](https://user-images.githubusercontent.com/33435621/57102824-31293d00-6cd9-11e9-92e5-3af14bf6eade.png)

![image](https://user-images.githubusercontent.com/33435621/57102860-3f775900-6cd9-11e9-8865-a2ee7670023d.png)

**Comment by ASprinkle** [comment]
@mtlaney can you check the AC boxes that have specific measures in them? I do not know how to quickly verify the 508 or the # for each color code...  I have checked the AC and task boxes that I can verify.

**Comment by mtlaney** [comment]
@ASprinkle Believe I checked correct boxes above. 
One note:  Tan background color #E2DCC9 (medium tan) - Title text is #333333 (dark grey), those exact colors were not used in keeping with the current colors of the site they were replaced with similar and existing colors.
